### ISSUE TITLE
Switch back to master fork of Mediawiki LDAPProvider

### ISSUE THREAD

**Issue by m1cr0man** [redbrick/nix-configs#21]
I have forked and extended LDAPProvider to add gidNumber resolution so we can use group-based privileges.

https://gerrit.wikimedia.org/r/#/c/mediawiki/extensions/LDAPProvider/+/559951/

When this is merged the LDAPProvider github fetch will need to be switched back.

**Comment by theycallmemac** [comment]
@m1cr0man has this been merged in?

**Comment by m1cr0man** [comment]
Nope, it was broken in the end. I need to submit another gerrit change.
